wordpress-mobile/WordPress-Android
# Add test-friendly AppLog

Fixes #8817 

Creates a test-friendly copy of _AppLog_ for tests in _WPAndroid_.
- test-friendly = without any Android dependencies


Removes TODO in the _NewSiteCreationService_ and moves the logging into _NewSiteCreationServiceManager_ so the new AppLog can be tested in _NewSiteCreationServiceManagerTest_.


To test:
- CI should take care of testing. 

Note: Update TODO in #9079 after merge. Thanks!

Update release notes:

- [ x ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.
